aleemer/Internify
# [IN-50] Connecting the Create pages together

#### User Story
[[IN-50]](https://chaosneutral.atlassian.net/jira/software/projects/IN/boards/1/backlog?selectedIssue=IN-50)

#### Changes made
- Connected the `Create` pages together via trigger activation on appropriate stepper count
- Updated `handleChange` and other hook events to update `jobData` values in each `Create` step
- Updated children props to have appropriate keys
- [Bonus] Updated favicon

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/37882255/123505051-be465300-d611-11eb-9384-a42cf36d54c9.png">


#### Does your new code introduce new warnings on dev console?
No

#### Test Steps
1. Go to the `/create` page
2. Fill in the forms and step through by clicking on the `continue` button at the bottom of the page
3. When viewing the final step (Review page), your information should be correctly displayed on the styled document
